plugin/trino-google-sheets/src/test/java/io/trino/plugin/google/sheets/TestGoogleSheets.java

```diff
@@ -97,11 +100,13 @@ private String createSpreadsheetWithTestdata()
         ), ImmutableList.of(
```

Comment:
Can you please fix indentation as I mentioned in other comment. I could go in separate commit as this unrelated to this change.